Wox.Infrastructure/Wox.cs

```diff
@@ -1,4 +1,3 @@
-using System;
 using System.Diagnostics;
```

Comment:
Although unrelated, but maybe we can change the file name from Wox to constant? :)